doc/Changes.md

```diff
@@ -0,0 +1,6343 @@
+# GAP - Changes from Earlier Versions
+
+Release 4.10.2, 19-Jun-2019
```

Comment:
The header could be perhaps polished and squeezed later, but for now at least this line is wrong.

Reply:
done